.travis.yml

```diff
@@ -2,10 +2,26 @@ language: erlang
 os: linux
 dist: trusty
 
-otp_release:
-   - 21.2.3
-   - 20.3.8.5
-   - 19.3
+matrix:
+  include:
+  - language: elixir
+    elixir: 1.6
+    env: COMMAND=elixir
+    otp_release: 21.2.3
+  - language: erlang
```

Comment:
Eventually, we're going to want elixir tests running against all OTP releases. I see this as a temporary workaround, but it's ugly. Using the `before_script` to install elixir, or just installing elixir in all versions, might be a better approach. See other comments as well.